src/js/utils/video-fit.js

```diff
@@ -0,0 +1,26 @@
+import { transform } from 'utils/css';
```

Comment:
utils is for common utils used across the code base.

This is code specific to providers. It belongs in the providers folder, next to video-actions-mixin.js or just put this code in that mixin.